apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -53,7 +53,8 @@ function showHomepage() {
               buttonText: i18n.learnMore(),
               description: i18n.announcementDescription(),
               link: "http://teacherblog.code.org/post/160703303174/coming-soon-access-your-top-resources-with-the",
-              image: "redesign-screencast"
+              image: "redesign-screencast",
```

Comment:
do we need to continue to pass in an image? or are you just keeping this here until we know 100% if we'll use Notifications or Announcements for announcements?